### DISCUSSION TITLE
Add config option to switch between the new and old command syntax

### DISCUSSION THREAD

**Discussion by wizard-28** [pacstall/website#17] [Ideas]
Issue created by @saenai255

**Comment by wizard-28** [comment]
Why do we need to do this? Why maintain backwards compatibility? We've already decided not to do so.

**Reply by Elsie19** [reply]
We aren't. We're just making it easy to switch to the new syntax

**Reply by wizard-28** [reply]
Why do people need to switch? Why isn't it the default?

**Reply by Elsie19** [reply]
I assume the website will be finished before the rewrite, so it would be nice to just have a variable that can switch to the rewrite syntax when it's released

**Reply by wizard-28** [reply]
So, the website will have it's own config file? Or will this be a toggle in the website itself?

**Reply by Elsie19** [reply]
The website config

**Reply by wizard-28** [reply]
So a file in this repository?

**Comment by Elsie19** [comment]
This has been completed for a long time.
